app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb

```diff
@@ -44,7 +44,8 @@ def add_intro
         "<font size='11'>Hypertension Rapid Ready for Decision | Claim for Increase</font>\n",
         "<font size='22'>VHA Hypertension Data Summary for</font>",
         "<font size='22'>#{patient_name}</font>\n",
-        "<font size='10'><i>Generated automatically on #{generated_at}<i>\n"
+        "<font size='10'><i>Generated automatically on #{generated_at}<i>\n",
```

Comment:
Does this line need a closing `</font>` tag?

Reply:
YES, thank you. I would love to find a way to use some kind of template for this stuff.